<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogpoubelles</class>
 <widget class="QDialog" name="Dialogpoubelles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>768</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::tab-bar {
   border: 1px solid gray;
}
QTabBar::tab {
  background: gray;
  color: white;
  padding: 10px;
 }

 QTabBar::tab:selected {
  background: lightgray;
 }
QTabWidget::pane { 
   border: none;
}</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter poubelle</string>
    </attribute>
    <widget class="QLabel" name="labelbackajout">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1366</width>
       <height>768</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>210</y>
       <width>591</width>
       <height>471</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Ajout Poubelle</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_adr">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid;
border-color: rgb(0,0,0);
border-radius: 25% 10%;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Exp:20,rue med</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_adress">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Adresse :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_long">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Longitude:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_long">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>180</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid;
border-color: rgb(0,0,0);
border-radius: 25% 10%;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Exp :50.246855</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_l">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>260</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Latitude:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_lat">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>250</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid;
border-color: rgb(0,0,0);
border-radius: 25% 10%;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="placeholderText">
       <string>Exp :10.875664</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_numcap">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid;
border-color: rgb(0,0,0);
border-radius: 25% 10%;</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
      <property name="placeholderText">
       <string>number:20</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_numcap">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Num capteur:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonenregistrer">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>390</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: grey;
</string>
      </property>
      <property name="text">
       <string>ENREGISTER</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAnnuler">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: grey;
</string>
      </property>
      <property name="text">
       <string>ANNULER</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_etat">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>330</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Etat:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_etat">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>320</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid;
border-color:rgb(0, 0, 0);
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
    color: rgb(220, 220, 220);
    background-color:rgb(89, 155, 255);
    margin: 0 0 0 0;</string>
      </property>
      <item>
       <property name="text">
        <string>Veuiller choisir</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vide</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pleine</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hors service</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Supprimer/Modifer poubelle</string>
    </attribute>
    <widget class="QLabel" name="labelbacsup">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1366</width>
       <height>768</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>210</y>
       <width>791</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Supprimer/modifier poubelle</string>
     </property>
     <widget class="QLabel" name="label_Id">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID poubelle</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Idsupmod">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid;
border-color: rgb(0,0,0);
border-radius: 25% 10%;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSupMod">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>330</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: grey;
</string>
      </property>
      <property name="text">
       <string>Supprimer/Modifer</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonSupprimerPou">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonModifierPou">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>70</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonValiderpou">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>62</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: grey;
</string>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Nouv_NumCap">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>NumCap</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNouvNumCap">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>149</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid;
border-color: rgb(0,0,0);
border-radius: 25% 10%;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNouvLongitude">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>150</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid;
border-color: rgb(0,0,0);
border-radius: 25% 10%;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_NouvLongitude">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Longitude </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_NouvLatitude">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Latitudel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNouvLatitude">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>239</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid;
border-color: rgb(0,0,0);
border-radius: 25% 10%;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNouvAdresse">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>240</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid;
border-color: rgb(0,0,0);
border-radius: 25% 10%;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_NouvAdresse">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>240</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Etatsupmod">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Etat </string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Etatsupmod">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(0, 0, 0);
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
    color: rgb(220, 220, 220);
    background-color:rgb(89, 155, 255);
    margin: 0 0 0 0;</string>
      </property>
      <item>
       <property name="text">
        <string>Vide</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pleine</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hors service</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Afficher poubelle</string>
    </attribute>
    <widget class="QLabel" name="labelbacmodif">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1366</width>
       <height>768</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>591</width>
       <height>441</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color:white;
</string>
     </property>
     <property name="title">
      <string>POUBELLE</string>
     </property>
     <widget class="QTableView" name="recherche_View">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>551</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 25% 10%;
background:grey;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_IdRech">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>ID POUBELLE</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idRech_le">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid color rgb(170, 85, 127);
border-radius: 25% 10%;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);

</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher_Pb">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>80</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid;
border-color:rgb(255, 255, 255);
background-color: grey;
</string>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>380</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Prestige Signature Script  Demo</family>
        <pointsize>30</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white</string>
      </property>
      <property name="text">
       <string>Med Ali Azouzi</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="tableView_Afficher">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>390</y>
       <width>571</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 25% 10%;
background:grey;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Tri">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>280</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;
border-color:rgb(0, 0, 0);
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
    color: rgb(220, 220, 220);
    background-color:rgb(89, 155, 255);
    margin: 0 0 0 0;</string>
     </property>
     <item>
      <property name="text">
       <string>id_poubelle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Num_capteur</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>etat</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_trierpar">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>280</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Trier par</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Type_Tri">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>280</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid;
border-color:rgb(0, 0, 0);
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
    color: rgb(220, 220, 220);
    background-color:rgb(89, 155, 255);
    margin: 0 0 0 0;</string>
     </property>
     <item>
      <property name="text">
       <string>croissant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>decroissant</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="Afficher_Pb">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>330</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid;
border-color:rgb(255, 255, 255);
background-color: grey;
</string>
     </property>
     <property name="text">
      <string>Afficher</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>210</y>
       <width>641</width>
       <height>441</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color:white;
</string>
     </property>
     <property name="title">
      <string>TRI POUBELLES</string>
     </property>
     <widget class="QPushButton" name="pushButtonStatistiques">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>380</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid;
border-color:rgb(255, 255, 255);
background-color: grey;
</string>
      </property>
      <property name="text">
       <string>STATISTIQUES</string>
      </property>
     </widget>
    </widget>
    <zorder>labelbacmodif</zorder>
    <zorder>groupBox_4</zorder>
    <zorder>tableView_Afficher</zorder>
    <zorder>comboBox_Tri</zorder>
    <zorder>label_trierpar</zorder>
    <zorder>Type_Tri</zorder>
    <zorder>Afficher_Pb</zorder>
    <zorder>groupBox_3</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
